docs/internals/data-structures.rst

```diff
@@ -278,24 +314,32 @@ each time an archive is added, modified or deleted.
 .. _archive:
 
 Archives
---------
-
-The archive metadata does not contain the file items directly. Only
-references to other objects that contain that data. An archive is an
-object that contains:
+~~~~~~~~
 
-* version
-* name
-* list of chunks containing item metadata (size: count * ~40B)
-* cmdline
-* hostname
-* username
-* time
+Each archive is an object referenced by the manifest. The archive object
+itself does not store any of the data contained in the archive it describes.
+
+Instead, it contains a list of chunks which form a msgpacked stream of items_.
+The archive object itself further contains some metadata:
+
+* *version*
+* *name*, which might differ from the name set in the manifest.
+  When :ref:`borg_check` rebuilds the manifest (e.g. if it was corrupted) and finds
+  more than one archive object with the same name, it adds a counter to the name
+  in the manifest, but leaves the *name* field of the archives as it was.
+* *items*, a list of chunk IDs containing item metadata (size: count * ~33B)
```

Comment:
Previously it said 40B, but that's actually not correct, because Archive.items is not a ChunkListEntry (id, size, csize), but only the IDs, which are raw8 in msgpack = 1 prefix byte + 1 length byte + 32 bytes = 34B.